src/ezdxf/tools/text.py

```diff
@@ -9,11 +9,13 @@
     Iterable,
     Iterator,
     TYPE_CHECKING,
+    Literal,
     Union,
     Optional,
     Callable,
     NamedTuple,
     Any,
+    overload,
```

Comment:
has to be imported from ` typing_extensions` , not available in Python 3.9 (requires 3.11)

Reply:
I believe you misread the documentation. `typing.overload` is the part of PEP 484.